doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -449,7 +449,135 @@ and inject the modified monitor map into each new monitor.
 Migration to the new location is now complete. The monitors should operate
 successfully.
 
+Using cephadm to change the public network
+==========================================
 
+Overview
+--------
+
+The procedure in this overview section provides only the broad outlines of
+using ``cephadm`` to change the public network.
+
+#. Create backups of all keyrings, configuration files, and the current monmap.
+
+#. Stop the cluster and disable ``ceph.target`` to prevent the daemons from
```

Comment:
I thought for sure we had something like  

https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3/html/administration_guide/understanding-process-managemnet-for-ceph#powering-down-and-rebooting-a-red-hat-ceph-storage-cluster-management

in upstream docs to which we could link here, but I'm not finding it.  Suggest that we add a section to doc/rados/operations/control.rst and link it here.

Reply:
I'll make a note to add this. Thanks, @anthonyeleven.